ci/release/update-version.sh

```diff
@@ -40,9 +40,9 @@ for FILE in conda/environments/*.yml; do
    sed_runner "s/rmm=${CURRENT_SHORT_TAG}/rmm=${NEXT_SHORT_TAG}/g" ${FILE};
    sed_runner "s/dask-cuda=${CURRENT_SHORT_TAG}/dask-cuda=${NEXT_SHORT_TAG}/g" ${FILE};
    sed_runner "s/dask-cudf=${CURRENT_SHORT_TAG}/dask-cudf=${NEXT_SHORT_TAG}/g" ${FILE};
-   sed_runner "s/ucx-py=${CURRENT_SHORT_TAG}/ucx-py=${NEXT_SHORT_TAG}/g" ${FILE};
+   sed_runner "s/ucx-py=0.20/ucx-py=0.20/g" ${FILE};
```

Comment:
I think this would end up updating ucx-py to 21.xx when the runner does its thing? @raydouglass might be able to comment more.

Reply:
@ajschmidt8 mentioned in https://github.com/rapidsai/cuml/pull/3950#discussion_r646670375 this would be a no-op. I'm not familiar with the use it's made here, but can we remove it @dantegd as AJ suggested?

Reply:
yup, sounds good to remove it